File: Views/src/components/out-sourced/collect-deliver/collect-deliver-create/dto/destiny-dto.ts
import { CustomerDto } from "src/components/main/customer/dtos/customer-dto";
import { PartnerDto } from "src/components/main/partner/dto/partner-dto";
import { CollectDeliverDto } from "./collect-deliver-dto";


export class DestinyDto {
  id: number;
  customerId: number;
  // customer: CustomerDto;
  partnerId: number;
  // partner: PartnerDto;
  noRegisterName: string;
  noRegisterAddress: string;
  description: string;
  collectDeliverId: number;
  // collectDeliver: CollectDeliverDto;
}

File: Application/Services/Operations/Outsorced/Dtos/DestinyDto.cs
using System;
using Application.Services.Operations.Main.Customers.Dtos;
using Application.Services.Operations.Main.Partners.Dtos;
using Domain.Entities;

namespace Application.Services.Operations.Outsourced.Dtos
{
    public class DestinyDto
    {
        public int Id { get; set; }
        public int? CustomerId { get; set; }
        public CustomerDto Customer { get; set; }
        public int? PartnerId { get; set; }
        public PartnerDto Partner { get; set; }
        public string NoRegisterName { get; set; }
        public string NoRegisterAddress { get; set; }
        public string Description { get; set; }
        public int CollectDeliverId { get; set; }
        public CollectDeliverDto CollectDeliver { get; set; }

    }
}
